appengine-skeleton-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -4,125 +4,123 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It would be nice to have a license here.   Surprisingly, the first line isn't required.